Lib/fontTools/varLib/merger.py

```diff
@@ -123,8 +123,12 @@ def mergeTables(self, font, master_ttfs, tableTags):
 		self.ttfs = master_ttfs  # For error reporting
 		for tag in tableTags:
 			if tag not in font: continue
-			self.mergeThings(font[tag], [m[tag] if tag in m else None
-						     for m in master_ttfs])
+			try:
+				self.mergeThings(font[tag], [m[tag] if tag in m else None
+							     for m in master_ttfs])
+			except Exception as e:
```

Comment:
should this only try to catch VarLibMergeError instead of bare Exception? at least for the former we know how these args are going to be handled and formatted